src/shell/shell_cmds.cpp

```diff
@@ -2506,3 +2508,151 @@ void DOS_Shell::CMD_MOVE(char* args)
 		}
 	}
 }
+
+static std::vector<std::string> search_files(const std::string_view query)
+{
+	const auto save_dta = dos.dta();
+	dos.dta(dos.tables.tempdta);
+	const auto dta = DOS_DTA(dos.dta());
+
+	auto found = DOS_FindFirst(std::string(query).c_str(),
+	                           ~(FatAttributeFlags::Volume |
+	                             FatAttributeFlags::Directory));
+
+	auto files = std::vector<std::string>();
```

Comment:
minor (style): We tend to use the `auto std::vector<blah>` = {}` form more.

Reply:
It's a small thing, but that would be preferable, thanks. We tend to use that style when we define a var of a specific type and initialise with defaults. Here's a small sampling, there's a lot more:

<img width="997" alt="image" src="https://github.com/dosbox-staging/dosbox-staging/assets/698770/dfa37cf4-c328-4711-8852-e422d2de3338">